markup-workflow.md

```diff
@@ -0,0 +1,29 @@
+ # Markup workflow
+
+Markup is the process which we take design sketch and convert them to Html/Css files. 
+The purpose of this chapter is to understand to process of creating markup with outsourcing providers.
+ 
+###Prepare the instruction file
+Create an explanation file contains the design sketch and instructions to the Markup guy (or girl).  
+you can see example [here](https://docs.google.com/spreadsheets/d/1GUch-TRll47WAbuLORqHeElqrEHolMTPg6kFnU97ItY/edit#gid=0).
```

Comment:
@noamoss @bricel 
I want to give an example of the instruction file, but I'm not sure if it is ok to use MX1 example (remember Gizra Way book is public).
What do you say?